data/src/main/java/com/bottlerocketstudios/brarchitecture/data/model/LinksDto.kt

```diff
@@ -7,12 +7,12 @@ import kotlinx.parcelize.Parcelize
 
 @JsonClass(generateAdapter = true)
 @Parcelize
-data class Links(
-    @Json(name = "avatar") val avatar: Link? = null
+data class LinksDto(
```

Comment:
@VanyaHuamanBottleRocket Go ahead and make all newly refactored Dto classes to extend the `Dto` interface

Reply:
Ah ok. I didn't even notice that interface.